src/network/networkd-bridge-mdb.c

```diff
@@ -252,30 +259,44 @@ static int bridge_mdb_verify(BridgeMDB *mdb) {
         if (section_is_invalid(mdb->section))
                 return -EINVAL;
 
-        if (mdb->family == AF_UNSPEC)
-                return log_warning_errno(SYNTHETIC_ERRNO(EINVAL),
-                                         "%s: [BridgeMDB] section without MulticastGroupAddress= field configured. "
-                                         "Ignoring [BridgeMDB] section from line %u.",
-                                         mdb->section->filename, mdb->section->line);
-
-        if (!in_addr_is_multicast(mdb->family, &mdb->group_addr))
-                return log_warning_errno(SYNTHETIC_ERRNO(EINVAL),
-                                         "%s: MulticastGroupAddress= is not a multicast address. "
-                                         "Ignoring [BridgeMDB] section from line %u.",
-                                         mdb->section->filename, mdb->section->line);
-
-        if (mdb->family == AF_INET) {
-                if (in4_addr_is_local_multicast(&mdb->group_addr.in))
+        if (mdb->type==BRIDGE_MDB_ENTRY_TYPE_L2){
+                if (!ether_addr_is_multicast(&mdb->l2_addr)){
+                        return log_warning_errno(SYNTHETIC_ERRNO(EINVAL),
+                                "%s: MulticastGroupAddress= is not an L2 multicast address. "
+                                "Ignoring [BridgeMDB] section from line %u.",
+                                mdb->section->filename, mdb->section->line);
+                }
```

Comment:
no {} around single-line if blocks, as per coding style

Reply:
(here and everywhere else)

Reply:
Agree, removed {}